common/scripts/lint-commits

```diff
@@ -31,6 +31,7 @@ for cmt in $("$GIT_CMD" rev-list --abbrev-commit $base..$tip)
 do
 	"$GIT_CMD" cat-file commit "$cmt" |
 	awk -vC="$cmt" '
+	(NR == 3) && /@users\.noreply\.github\.com>/ { print C ": commit author has a github noreply email" }
```

Comment:
Notwithstanding other non-technical questions. `NR == 3` sounds incorrect.
In my opinion, it should be
```suggestion
	/^author .*noreply/ { print C ": commit author has a noreply email" }
```

Reply:
Thanks, I modified your suggestion a bit to fix some potential false positives